gitbutler-ui/src/lib/components/ActiveBranchStatus.svelte

```diff
@@ -11,6 +13,22 @@
 	export let isUnapplied = false;
 	export let hasIntegratedCommits = false;
 	export let isLaneCollapsed: Persisted<boolean>;
+
+	function updateContextMenu() {
+		if (popupMenu) popupMenu.$destroy();
+		return new ViewPrContextMenu({
+			target: document.body,
+			props: { prUrl: prUrl || '' }
+		});
+	}
+
+	$: popupMenu = updateContextMenu();
```

Comment:
I haven't tested this out, but it looks like we ought to pass `prUrl` as a parameter to this function to make it react to new values?

Reply:
Thanks for pointing that out - I think you're probably right